src/Markdig/MarkdownExtensions.cs

```diff
@@ -59,6 +60,7 @@ public static MarkdownPipelineBuilder UseAdvancedExtensions(this MarkdownPipelin
                 .UseListExtras()
                 .UseTaskLists()
                 .UseDiagrams()
+                .UseNoRefLinks()
```

Comment:
It should not be added to the Advanced extension, as this one is supposed to be somewhat CommonMark compliant and should not change the default behaviour of links